source/agora/utils/Log.d

```diff
@@ -201,11 +201,11 @@ public class CircularAppender : Appender
 /// A layout with colored LogLevel
 public class AgoraLayout : Appender.Layout
 {
+    import ocean.time.WallClock;
```

Comment:
`Use agora's logger instead of stdio` Is it related to the correction?

Reply:
Yes, see the commit message:
```
Additionally, an import was moved because `dmd` didn't
pick it up when emitting dependencies for `rdmd`
and thus the tests in `test/unit/` had linker failures.
```

Reply:
Oh, thank you.  I understood.